frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -121,6 +115,7 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
           <LogsTabStatus
             loaded={loaded}
             error={error}
+            isLogsNotAvailable={logs === null || logs === ''}
```

Comment:
type for `logs` is a `string`. Doesn't say it can be `null`. Is the type incorrect?

We always want to treat `null` and `undefined` as the same value unless there's a specific reason not to.
Here i think you can simplify this to:
```suggestion
            isLogsNotAvailable={!logs}
```